example/README.md

```diff
@@ -49,6 +49,23 @@ This command will update the lambda with the latest lambda code.
 
 Test data comes from the @cumulus/test-data package and is uploaded to S3 during the setup step when running all tests. The data will be uploaded to the S3 bucket specified in the test configuration.
 
+### Fake data server
```

Comment:
If there's a restriction on the number of fakeServers that can be deployed this way, I think it would be a good idea to make that explicit here.